lib/src/transaction_builder.dart

```diff
@@ -10,6 +12,8 @@ import 'payments/p2pkh.dart';
 import 'payments/p2wpkh.dart';
 import 'classify.dart';
 
+const int MAX_OP_RETURN_SIZE = 256;
```

Comment:
This should rather be part of the NetworkType model as different coins have different OP_RETURN sizes.

Reply:
fixed